src/DynamoCoreWpf/Views/Preview/WatchTree.xaml

```diff
@@ -54,10 +54,144 @@
+                                                                               Storyboard.TargetName="Collapsed">
+                                                    <DiscreteObjectKeyFrame KeyTime="0"
+                                                                            Value="{x:Static Visibility.Hidden}" />
+                                                </ObjectAnimationUsingKeyFrames>
+                                                <ObjectAnimationUsingKeyFrames Storyboard.TargetProperty="(UIElement.Visibility)"
+                                                                               Storyboard.TargetName="Expanded">
+                                                    <DiscreteObjectKeyFrame KeyTime="0"
+                                                                            Value="{x:Static Visibility.Visible}" />
+                                                </ObjectAnimationUsingKeyFrames>
+                                            </Storyboard>
+                                        </VisualState>
+                                        <VisualState x:Name="Unchecked" />
+                                        <VisualState x:Name="Indeterminate" />
+                                    </VisualStateGroup>
+                                </VisualStateManager.VisualStateGroups>
+                                <Path x:Name="Collapsed"
+                                      HorizontalAlignment="Left"
+                                      VerticalAlignment="Center"
+                                      Margin="1,1,1,1"
+                                      Data="M 4 0 L 8 4 L 4 8 Z">
+                                    <Path.Fill>
+                                        <SolidColorBrush Color="{DynamicResource GlyphColor}" />
+                                    </Path.Fill>
+                                </Path>
+                                <Path x:Name="Expanded"
+                                      HorizontalAlignment="Left"
+                                      VerticalAlignment="Center"
+                                      Margin="1,1,1,1"
+                                      Data="M 0 4 L 8 4 L 4 8 Z"
+                                      Visibility="Hidden">
+                                    <Path.Fill>
+                                        <SolidColorBrush Color="{DynamicResource GlyphColor}" />
+                                    </Path.Fill>
+                                </Path>
+                            </Grid>
+                        </ControlTemplate>
+                    </Setter.Value>
+                </Setter>
+            </Style>
+
             <Style TargetType="{x:Type TreeViewItem}">
-                <Setter Property="IsExpanded" Value="{Binding IsNodeExpanded}">
+                <Setter Property="IsExpanded"
+                        Value="{Binding IsNodeExpanded}" />
+                <Setter Property="Template">
```

Comment:
I had to rewrite this default TreeViewItem Template in order to get rid of this margin. Actually it's hidden expander, which should be collapsed.
![image](https://cloud.githubusercontent.com/assets/8158404/11376492/acf401a4-92e9-11e5-83d1-a2bc5fc9afb7.png)
![image](https://cloud.githubusercontent.com/assets/8158404/11376503/bc3fb338-92e9-11e5-981c-743dea161ae7.png)